### DISCUSSION TITLE
Sentinel-1 RTC LIA

### DISCUSSION THREAD

**Discussion by Kongstad** [microsoft/PlanetaryComputer#279] [Data request]
Hi,

I am currently testing the MSPC Sentinel1-RTC for implementation into existing pipelines. The prospect of it replacing our current processing intensive SNAP engine pipeline is quite enticing. However I can't seem to retrieve the LIA "band". The Local Incidence Angle, which I believe is part of the RTC process. Is it only the VV, VH band that is provided and if so, how does everyone else go about with that? Just not use it or extract it in other ways. Perhaps its not that widely used. 

Thanks for your feedback.

**Comment by TomAugspurger** [comment]
Sorry for the delay @Kongstad. It sounds like it'd be technically feasible to transport LIA through the RTC process, but we don't currently provide that. We'll post here if that changes in the future.

**Reply by Kongstad** [reply]
Thank you for your reply.

**Comment by scottyhq** [comment]
@kongstad you might find this utility and documentation helpful https://github.com/egagli/generate_sentinel1_local_incidence_angle_maps

**Reply by Kongstad** [reply]
This is excellent! Thank you!
